docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
@bparees what would we be the use case for using a cacert in conjunction with an ssh private key?